### ISSUE TITLE
Windows release link wrong on main site

### ISSUE THREAD

**Issue by xymox12** [GuitarML/Chameleon#5]
Just a FYI that your link to the Windows release of the Chameleon amp is broken on the main site.

PS Great work by the way. Amazingly easy to train the SmartAmp Pro  - although at the moment the plugin freezes Reaper's interface. Shame my training is only getting around 79% accuracy so I need to work out how to improve that.

**Comment by GuitarML** [comment]
Thanks for letting know! It should be fixed on the website now.

If you want to Dropbox me the wav samples you're using to train I can see if I can get better accuracy. What amp/pedal are you using? SmartAmpPro has the fastest training of the machine learning models I use, but at the cost of accuracy.

**Comment by xymox12** [comment]
Great thanks :) 

Afraid I sold my guitar kit years ago, but because of lockdown decided to try my hand at music again via a DAW, and purchased a few virtual guitar instruments. So I need to hack test wav files as I am really just exploring the training at the moment. I went for the cheapest (in time) way of using Reaper to create a mono track of a 'virtual' guitar playing clean, and then with its amp applied with the pans set to left and right for the two mono instruments. Then rendered out to a wav and used Colab. 
I tried a similar experiment using white noise on the track instead of the the guitar, and applied a free amp to the right panned track. That scored 61%. Not that the json it produced didn't actually give an alright sound via smartamp pro - just not the same.
